submissions/evgenii-del/js-movie-seat-booking/index.js

```diff
@@ -0,0 +1,92 @@
+const closePopupBtn = document.querySelector('.js-popup__close');
+const popup = document.querySelector('.js-popup');
+const popupBtn = document.querySelector('.js-popup__btn');
+const overlay = document.querySelector('.js-overlay');
+const form = document.forms[0];
+
+const openPopup = () => {
+  popup.classList.add('popup_active');
+  overlay.classList.add('overlay_active');
+};
+
+const closePopup = () => {
+  popup.classList.remove('popup_active');
+  overlay.classList.remove('overlay_active');
+};
+
+const createTicketsBlock = (seats) => {
+  const fragment = document.createDocumentFragment();
+  seats.map((seat) => {
+    const [row, column] = seat.split('-');
+    const block = document.createElement('div');
+    block.className = 'popup__ticket';
```

Comment:
For manipulating classes use classList API [Read more](https://developer.mozilla.org/en-US/docs/Web/API/Element/classList).